ci/tasks/test-put.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+apk --no-progress --no-cache add tree
```

Comment:
I think you should remove these lines below (cannot comment on them):
```sh
# The test in job test-get will use `snapshot` to assert no diffs are present.
snapshot=`tree out`
echo $snapshot > out/snapshot
```

Because they are a lie now that `test-get` is removed.

Reply:
Done as suggested. Please have a look. Thanks.